.circleci/config.yml

```diff
@@ -7,17 +7,23 @@ shared: &shared
     - run:
         name: Install codebase
         command: cp -a /home/circleci/codebase/. ./
+    - restore_cache:
+        keys:
+          - composer-cache-{{ .Environment.CIRCLE_JOB }}
```

Comment:
This will load the last saved cache having this exact key or having a key that starts with this.